### ISSUE TITLE
Tiles / SDL not building anymore.

### ISSUE THREAD

**Issue by AlexTes** [CleverRaven/Cataclysm-DDA#12728]
Went from bc3d389 -> d53f969

Failed with:
```
src/sdltiles.cpp: In function ‘void curses_drawwindow(WINDOW*)’:
src/sdltiles.cpp:663:9: error: ‘g’ was not declared in this scope
     if (g && win == g->w_terrain && use_tiles) {
         ^
src/sdltiles.cpp:663:37: error: ‘use_tiles’ was not declared in this scope
     if (g && win == g->w_terrain && use_tiles) {
                                     ^
src/sdltiles.cpp: In member function ‘bool Font::draw_window(WINDOW*, int, int)’:
src/sdltiles.cpp:775:25: error: ‘g’ was not declared in this scope
             if ( win == g->w_terrain || win == g->w_minimap || win == g->w_HP || win == g->w_status ||
                         ^
src/sdltiles.cpp: In function ‘WINDOW* curses_init()’:
src/sdltiles.cpp:1448:9: error: ‘use_tiles’ was not declared in this scope
         use_tiles = false;
         ^
src/sdltiles.cpp: In function ‘void rescale_tileset(int)’:
src/sdltiles.cpp:1658:9: error: ‘g’ was not declared in this scope
         g->init_ui();
         ^
src/sdltiles.cpp: In member function ‘bool input_context::get_coordinates(WINDOW*, int&, int&)’:
src/sdltiles.cpp:1669:23: error: ‘g’ was not declared in this scope
         capture_win = g->w_terrain;
                       ^
src/sdltiles.cpp:1678:9: error: ‘use_tiles’ was not declared in this scope
     if (use_tiles && capture_win == g->w_terrain) {
         ^
src/sdltiles.cpp:1678:37: error: ‘g’ was not declared in this scope
     if (use_tiles && capture_win == g->w_terrain) {
                                     ^
src/sdltiles.cpp:1710:13: error: ‘g’ was not declared in this scope
         x = g->ter_view_x - ((capture_win->width / 2) - selected_column);
             ^
src/sdltiles.cpp:1718:13: error: ‘g’ was not declared in this scope
         x = g->ter_view_x - ((capture_win->width / 2) - selected_column);
             ^
src/sdltiles.cpp: In function ‘int map_font_width()’:
src/sdltiles.cpp:1908:9: error: ‘use_tiles’ was not declared in this scope
     if (use_tiles && tilecontext != NULL) {
         ^
src/sdltiles.cpp: In function ‘int map_font_height()’:
src/sdltiles.cpp:1917:9: error: ‘use_tiles’ was not declared in this scope
     if (use_tiles && tilecontext != NULL) {
         ^
src/sdltiles.cpp: In function ‘bool is_draw_tiles_mode()’:
src/sdltiles.cpp:1948:12: error: ‘use_tiles’ was not declared in this scope
     return use_tiles;
            ^
Makefile:454: recipe for target 'obj/tiles/sdltiles.o' failed
make: *** [obj/tiles/sdltiles.o] Error 1
```

**Comment by AlexTes** [comment]
Found the problem. Removing the "game.h" include in cataclysm_tiles is the problem.
See 2e97ce0.

**Comment by AlexTes** [comment]
Decided to try and fix the issue myself. Never done pull requests to public repo's, but I figured now is as good a time as any to start. It saves pretty much no work at all, but if I can do anything back to the people  who are the reason I am able to thoroughly enjoy this game, why not?!

**Comment by slyscorpion** [comment]
I just had this happen as well. So this means my game won't run now?

**Comment by AlexTes** [comment]
It means it won't build. You can still play what you have build previously, grab the last version of Cataclysm that did build, so right before 2e97ce0 over at http://dev.narc.ro/cataclysm/jenkins-latest/ , and finally you could also use git to revert that commit (might break other stuff) or checkout any commit prior to 2e97ce0 (git checkout 2e97ce0~1) and then build, or apply my change. Which is literally one line, added to one file, someone probably accidently removed during a cleanup. Click the "re-include (accidental) removal of 'game.h' fixes #12728" to see what one line you need to edit to get it to build again.

**Comment by slyscorpion** [comment]
Yeah I just went with the one line change myself :)
